prometheus/metrics.go

```diff
@@ -41,6 +43,8 @@ func (q *MetricsQuery) FillDefaults() {
 	q.Step = 15 * time.Second
 	q.RateInterval = "1m"
 	q.RateFunc = "rate"
+	q.Avg = true
+	q.Quantiles = []string{"0.5", "0.95", "0.99", "0.999"}
```

Comment:
This doesn't match the documented default.  Are you sure you want "0.999"?

Reply:
Oops, you're right (and doc is right, i'll remove 0.999 here)

Reply:
fixed. BTW I'm adding DNM label to sync with kiali/kiali-ui#701 review